.gitignore

```diff
@@ -1,3 +1,4 @@
 AngularApp/node_modules
 TestFramework/node_modules
-ReactApp/node_modules
\ No newline at end of file
+ReactApp/node_modules
+ReactLatestApp/node_modules
```

Comment:
@Vinay-Sunny add new line at end of file.. so that we will not see the red arrow alert on github